### DISCUSSION TITLE
Configuring ZOAU after initial installation on z/OS 2.4

### DISCUSSION THREAD

**Discussion by gngrossi** [IBM/zoau-samples#53] [Q&A] [answered]
[zoautil_configure.txt](https://github.com/IBM/zoau-samples/files/9966298/zoautil_configure.txt)
[zoautil-py_install.txt](https://github.com/IBM/zoau-samples/files/9966304/zoautil-py_install.txt)

Running into errors when trying to "configure" ZOAU via the pip method or the PYTHONPATH method.
Attached are two files, one from each attempt.

After the pip method, I was able to import the module even though it looked like it failed.

$ python3
Python 3.10.5 (heads/pyz_dev-3.10:613692d6dd, Jun 16 2022, 12:29:24) [Clang 4.0.1 (tags/RELEASE_401/final)] on zos
Type "help", "copyright", "credits" or "license" for more information.
>>> from zoautil_py import jobs
>>> jobs.listing("//ABC*")
[]

I was expecting a "clean" customization. Not sure what I'm missing.
Thanks,
Gary

**Comment by AnthonyGiorgio** [comment, marked as answer]
There was a packaging error with the ZOAU v1.2.1 release, which has been corrected with the v1.2.1.1 bugfix release.  

Also, if your environment is not setup such that you can execute python scripts directly, you can do `python3 configure.py` instead.

**Comment by gngrossi** [comment]
This week I downloaded both python and zoau. 
$ zoaversion
2022/10/29 02:46:39 CUT V1.2.1.1 b902f4b 2169

For our POC, we didn't complete this step. Is this the setup you're referring to?
ln -s /bin/env /usr/bin/env

thanks

**Comment by gngrossi** [comment]
Looks good running configure.py after updating the shebang from  #!/usr/bin/env python3  to  #!/bin/env python3
$ ./configure.py
ZOAU release built on python version: 3.9
The ZOAU python build version differs from the detected python version.
A symbolic link is being created to complete the ZOAU installation.
It will be located in: /hewitt/zopentools/pyz/usr/lpp/IBM/cyp/v3r10/pyz/lib
Python library name is: libpython3.10.so
Python library symlink will be: libpython3.9.so
Removing prior symlink: libpython3.9.so
Creating symlink: libpython3.9.so -> libpython3.10.so
ZOAU python configuration complete.
--
Afterwards, I uninstalled zoautil-py (it was installed without using a virtual env).
$ pip3 uninstall zoautil-py
Found existing installation: zoautil-py 1.2.1.1
Uninstalling zoautil-py-1.2.1.1:
  Would remove:
    /newco/zopentools/pyz/usr/lpp/IBM/cyp/v3r10/pyz/lib/python3.10/site-packages/zoautil_py
    /newco/zopentools/pyz/usr/lpp/IBM/cyp/v3r10/pyz/lib/python3.10/site-packages/zoautil_py-1.2.1.1-py3.10.egg-info
Proceed (Y/n)? Y
  Successfully uninstalled zoautil-py-1.2.1.1

thanks
